src/database/initialization/organization_to_json.sql

```diff
@@ -1,11 +1,44 @@
-CREATE OR REPLACE FUNCTION organization_to_json(organization organizations)
+-- An explicit DROP is needed to remove the old function instead of overloading it.
+DROP FUNCTION IF EXISTS organization_to_json(organizations);
```

Comment:
I was thinking about this and think we should plan to make a more universal solution (not part of this PR!)

This describes a solution, which basically is a function that will find all functions with a name (regardless of parameters), and drop each of them: https://stackoverflow.com/questions/7622908/drop-function-without-knowing-the-number-type-of-parameters

I think this should be applied to every initialization script since we don't want them to have to keep track of past state.

I'll make an issue for it!